### DISCUSSION TITLE
讨论技术信息

### DISCUSSION THREAD

**Discussion by ghost** [WeNeedHome/SummaryOfLoanSuspension#1002] [General]
大家可以在这里讨论技术信息。

**Comment by MyraToMoon** [comment]
加油，最近事情闹得挺大的，你们没有被抓起来的吧

**Comment by TALK1949** [comment]
奇怪，楼主昨天这帖子还是反应某地烂尾楼，包括停贷通知和视频的网盘链接，怎么今天标题内容也改了，链接也删了？
不会是被网警盯上了吧，大兄弟保重啊……
